### ISSUE TITLE
Show effect of stat choices at the time you pick them?

### ISSUE THREAD

**Issue by G0rg0** [jeremygurr/dcssca#645]
Would it be possible to have a screen that shows you what each stat choice will do? Basically the same as what you get when you quaff an XP potion.

It wouldn't necessarily need to be forced, you could just have a "?" option that would take you to said screen and let you see the effect of STR/DEX/INT on your char before you pick.

**Comment by Sandman25DCSS** [comment]
I think it is very hard to implement because every stat affects very many things and depends on your Armour also.

**Comment by jeremygurr** [comment]
Yes, there are many effects, but I think we can at least start with the easy ones, and maybe give vague descriptions for the rest:

Strength:
increases damage. (e.g. increases your spear damage from 3.2 to 3.4)
reduces your spell failure when wearing heavy armour. (e.g. fire storm goes from 55% to 53% failure)

Intelligence:
increases spellpower.
reduces spell cost. (e.g. your fire storm spell goes from 32 to 30)
reduces spell failure.

Dexterity:
reduces spell failure. (e.g. your fire storm fail rate goes from 55% to 52%)
increases accuracy. (e.g. your bow goes from 75% hit chance to 78% hit chance against goblin)
